app/services/clients/products_client.js

```diff
@@ -91,9 +96,29 @@ function getProductsByGatewayAccountId (gatewayAccountId) {
   }).then(products => products.map(product => new Product(product)))
 }
 
+/**
+ * @param {String} productExternalId: the external id of the product you wish to retrieve
+ * @returns {Promise<Product>}
+ */
+function updateProductServiceName (productExternalId, serviceName) {
```

Comment:
I don't see a need for this. I cannot `pay-products-ui` ever need to update the service name. It should always come from `selfservice`

Reply:
I've added this code only in the interests of keeping the products clients in both project identical.